app/src/main/java/com/vero/tasky/agenda/data/remote/network/api/EventApi.kt

```diff
@@ -0,0 +1,55 @@
+package com.vero.tasky.agenda.data.remote.network.api
+
+import com.vero.tasky.agenda.data.remote.network.dto.AttendeeCheckDto
+import com.vero.tasky.agenda.data.remote.network.dto.EventDto
+import okhttp3.MultipartBody
+import retrofit2.http.DELETE
+import retrofit2.http.GET
+import retrofit2.http.Multipart
+import retrofit2.http.POST
+import retrofit2.http.PUT
+import retrofit2.http.Part
+import retrofit2.http.Query
+
+interface EventApi {
+
+    @Multipart
+    @POST("event")
+    suspend fun createEvent(
+        @Part id: MultipartBody.Part,
+        @Part title: MultipartBody.Part,
+        @Part description: MultipartBody.Part,
+        @Part from: MultipartBody.Part,
+        @Part to: MultipartBody.Part,
+        @Part remindAt: MultipartBody.Part,
+        @Part attendeeIds: List<MultipartBody.Part>,
```

Comment:
There should only be 2 parts for this request.
1. The JSON serialized body which represents the created event
2. The photos